Sources/Plasma/PubUtilLib/plPipeline/plTextFont.cpp

```diff
@@ -136,23 +157,57 @@ uint16_t  *plTextFont::IInitFontTexture()
     // Loop through characters, drawing them one at a time
     RECT    r;
     r.left = r.top = 0;
-    r.right = r.bottom = 10;
+    r.right = 1;
+    r.bottom = 10;
     FillRect( hDC, &r, (HBRUSH)GetStockObject( GRAY_BRUSH ) );
 
     // (Make first character a black dot, for filling rectangles)
     SetPixel( hDC, 0, 0, RGB( 255, 255, 255 ) );
```

Comment:
Does anyone know what lines 158 through 165 are about? The first character is going to be a space, which feels sufficient. I have no idea what this other code is doing. @dpogue @Hoikas

Reply:
That looks a little bit like it's drawing a cursor, assuming you mean:
```C++
    RECT    r;
    r.left = r.top = 0;
    r.right = r.bottom = 10;
    FillRect( hDC, &r, (HBRUSH)GetStockObject( GRAY_BRUSH ) );

    // (Make first character a black dot, for filling rectangles)
    SetPixel( hDC, 0, 0, RGB( 255, 255, 255 ) );
```
Not sure about the `SetPixel()` call though.